website/templates/include/profile/social.mako

```diff
@@ -58,8 +58,8 @@
             <div class="form-group">
                 <label>LinkedIn</label>
                 <div class="input-group">
-                <span class="input-group-addon">https://www.linkedin.com/profile/view?id=</span>
-                <input class="form-control" data-bind="value: linkedIn" placeholder="profileID"/>
+                <span class="input-group-addon">https://www.linkedin.com/</span>
+                <input class="form-control" data-bind="value: linkedIn" placeholder="path to UserID or profileID"/>
```

Comment:
It is more effective to use an example rather than a description (like the other placeholders in the form).

Something like:

```
/in/userID or profile/view?id=xxxxxxxxx
````